lib/cloud_controller/db_migrator.rb

```diff
@@ -13,6 +13,9 @@ def self.from_config(config, db_logger)
   def initialize(db, max_migration_duration_in_minutes=nil)
     @db = db
     @timeout_in_minutes = default_two_weeks(max_migration_duration_in_minutes)
+    # Set serverside timeouts(30s) to protect against badly written migrations and prevent downtimes
```

Comment:
I'm not sure if we should hard-code this or have a config property instead...

Reply:
yea me too this is just a draft not yet finished or rdy for review :)